src/services/config/defaults.ts

```diff
@@ -0,0 +1,80 @@
+import fs from "fs-extra";
+import type { Context } from "../command/context.js";
+import { swallowEnoent } from "../filesync/directory.js";
+import { output } from "../output/output.js";
+import { println } from "../output/print.js";
+import { select } from "../output/select.js";
+import { config } from "./config.js";
+
+export type DefaultsConfigData = {
+  telemetry?: boolean;
+  json?: boolean;
+};
+
+export const promptDefaultsConfig = async (ctx: Context): Promise<DefaultsConfigData> => {
+  const path = config.defaultsConfigFile;
+  await fs.ensureFile(path);
+
+  println({ ensureEmptyLineAbove: true, content: "Please answer the prompts to configure ggt defaults." });
+
+  const telemetrySelection = await select({
+    ensureEmptyLineAbove: true,
+    choices: ["enable", "disable"],
+    content: "Would you like to automatically send crash reports and telemetry to Gadget?",
```

Comment:
I'd rephrase this to `Automatically send crash reports and telemetry to Gadget.`